sdk/python/feast/feature_store.py

```diff
@@ -1017,6 +1013,30 @@ def _augment_response_with_on_demand_transforms(
                     ] = GetOnlineFeaturesResponse.FieldStatus.PRESENT
         return OnlineResponse(GetOnlineFeaturesResponse(field_values=result_rows))
 
+    def _get_feature_views_to_use(
+        self,
+        features: Optional[Union[List[str], FeatureService]],
```

Comment:
At which point do we pass a `FeatureService` to this method?

Reply:
in both get_historical_features and get_online_features